types/cronitor/index.d.ts

```diff
@@ -0,0 +1,113 @@
+type MonitorState = "run" | "complete" | "fail" | "ok";
```

Comment:
If you want these types to be available to consumers, you can wrap everything in a `declare namespace Cronitor` block (the namespace name should be the same as the thing being exported with `export =`).

The only other change that would need to be made would be changing line 111 to reference `Cronitor.CronitorConfig` instead